tests/kernel/wave/wave_gemm_test.py

```diff
@@ -219,6 +227,299 @@ def repeat(acc: tkl.Register[M, N, tkl.f32]) -> tkl.Register[M, N, tkl.f32]:
 
 
 @require_e2e
+@require_cdna2
```

Comment:
This is cool. We should talk to Sai and try to get a MI250 builder to verify our CDNA2 tests.

Reply:
that'll be very nice! Initially, I added that flag S.T the cdna3 exclusive tests wouldn't fail when @Giuseppe5 is running on his machine haha.

Reply:
done, I think! :)